### ISSUE TITLE
Postgres doesn't start

### ISSUE THREAD

**Issue by deimosfr** [oznu/docker-unms#35]
Hi,

I've got a problem running 0.13.2 and got issue with postgres:
```
s6-setuidgid postgres postgres                                                                                              
s6-applyuidgid: fatal: unable to exec postgres: No such file or directory
```

I could make it running with a dirty hack:
```
ln -s /usr/lib/postgresql/9.6/bin/postgres /usr/bin/
```

But it would be better to fix it properly by adding the bin path in the PATH.

Thanks

**Comment by ofadam** [comment]
How exactly would I adjust the PATH to fix it? Just add ":/usr/lib/postgresql/9.6/bin/"?

**Comment by deimosfr** [comment]
It works thanks
